tst/testinstall/recordname.tst

```diff
@@ -71,4 +71,97 @@ Syntax error: Identifiers in GAP must consist of at most 1023 characters. in s\
 tream:2
 aaaxyz := 1; # 1024 chars
  ^^^^^
+gap> r := rec(y := 2);
+rec( y := 2 )
+gap> \.(r, RNamObj("y"));
+2
+gap> \.(r, RNamObj("x"));
+Error, Record Element: '<rec>.x' must have an assigned value
+gap> \.(r, 0);
+Error, Record Element: <rnam> must be a positive small integer (not the intege\
+r 0)
+gap> \.(r, -1);
+Error, Record Element: <rnam> must be a positive small integer (not the intege\
+r -1)
+gap> \.(r, 2^1000);
+Error, Record Element: <rnam> must be a positive small integer (not a large po\
+sitive integer)
+gap> \.(r, "a");
+Error, Record Element: <rnam> must be a positive small integer (not a list (st\
+ring))
+gap> \.(r, 1000000);
+Error, Record Element: <rnam> must be a valid rnam (not the integer 1000000)
+
+##
+gap> IsBound\.(r, RNamObj("y"));
+true
+gap> IsBound\.(r, RNamObj("x"));
+false
+gap> IsBound\.(r, 0);
+Error, Record IsBound: <rnam> must be a positive small integer (not the intege\
+r 0)
+gap> IsBound\.(r, -1);
+Error, Record IsBound: <rnam> must be a positive small integer (not the intege\
+r -1)
+gap> IsBound\.(r, 2^1000);
+Error, Record IsBound: <rnam> must be a positive small integer (not a large po\
+sitive integer)
+gap> IsBound\.(r, "a");
+Error, Record IsBound: <rnam> must be a positive small integer (not a list (st\
+ring))
+gap> IsBound\.(r, 1000000);
+Error, Record IsBound: <rnam> must be a valid rnam (not the integer 1000000)
+
+##
+gap> r;
+rec( y := 2 )
+gap> Unbind\.(r, RNamObj("y"));
+gap> r;
+rec(  )
+gap> Unbind\.(r, RNamObj("y"));
+gap> r;
+rec(  )
+gap> Unbind\.(r, RNamObj("x"));
+gap> Unbind\.(r, 0);
+Error, Record Unbind: <rnam> must be a positive small integer (not the integer\
+ 0)
+gap> Unbind\.(r, -1);
+Error, Record Unbind: <rnam> must be a positive small integer (not the integer\
+ -1)
+gap> Unbind\.(r, 2^1000);
+Error, Record Unbind: <rnam> must be a positive small integer (not a large pos\
+itive integer)
+gap> Unbind\.(r, "a");
+Error, Record Unbind: <rnam> must be a positive small integer (not a list (str\
+ing))
+gap> Unbind\.(r, 1000000);
+Error, Record Unbind: <rnam> must be a valid rnam (not the integer 1000000)
+
+##
+gap> r;
+rec(  )
+gap> \.\:\=(r,RNamObj("y"), 2);
```

Comment:
You could also use `ASS_REC` instead of `\.\:\=`, that might be a bit easier to grok. Same for `ISB_REC`, `ELM_REC`, `UNB_REC`, of course, though there it's perhaps less of a concern.

Reply:
I'd prefer to use \.\:\= and friends, are those are what we actually document.